README.md

```diff
@@ -26,4 +54,4 @@ To test in a local Jenkins instance
 
     mvn hpi:run
 
-  [wiki]: https://wiki.jenkins-ci.org/display/JENKINS/Plain+Credentials+Plugin
+  [wiki]: https://plugins.jenkins.io/plain-credentials
```

Comment:
Probably not needed link anymore, you can view docs either on plugin site or GitHub

Reply:
Yeah removed :+1